ImageSegmentation/deeplabv3.py

```diff
@@ -0,0 +1,8 @@
+import torch
+
+model = torch.hub.load('pytorch/vision:v0.7.0', 'deeplabv3_resnet50', pretrained=True)
+model.eval()
+
+scriptedm = torch.jit.script(model)
+torch.jit.save(scriptedm, "deeplabv3_scripted.pt")
```

Comment:
maybe to store it directly to assets folder?
```
torch.jit.save(scriptedm, "app/src/main/assets/deeplabv3_scripted.pt")
```

Reply:
yes yours is better. but i'll do it in the future when updating the accompanying tutorial and the readme as this is mentioned in a few places.